sstables/sstables_manager.cc

```diff
@@ -171,7 +171,9 @@ void sstables_manager::increment_total_reclaimable_memory_and_maybe_reclaim(ssta
     _total_memory_reclaimed += memory_reclaimed;
     _total_reclaimable_memory -= memory_reclaimed;
     _reclaimed.insert(*sst_with_max_memory);
-    smlogger.info("Reclaimed {} bytes of memory from SSTable components. Total memory reclaimed so far is {} bytes", memory_reclaimed, _total_memory_reclaimed);
+    // TODO: As of now only bloom filter is reclaimed. Print actual component names when adding support for more components.
+    smlogger.info("Reclaimed {} bytes of memory from components of {}. Total memory reclaimed so far is {} bytes",
+            sst_with_max_memory->get_filename(), memory_reclaimed, _total_memory_reclaimed);
```

Comment:
Won't this put the sstable's path in the `Reclaimed {} bytes`?

Reply:
my bad. I'll update it.

Reply:
fixed it now.